content/admin/teamserver/config/integration/SSO.md

```diff
@@ -0,0 +1,99 @@
+<!--
+title: "Configuring Single Sign-On (SSO)"
+description: "TeamServer can be configured to utilize Single Sign On Authentication"
+tags: "SSO Single Sign-On configuration authentication"
+-->
+
+## Getting Started
+
+Single Sign-On (SSO) is an authentication service that allows access to multiple applications using one set of credentials. Contrast can now be configured to use this service.
+
+Authentication happens via an **Identity Provider (IdP)**. You may use either your own generic IdP or one of many popular third-party providers, such as [Okta](https://www.okta.com/), [OneLogin](https://www.onelogin.com/) or [Ping Identity](https://www.pingidentity.com/en.html), to name a few. <!--(Add hyperlinks to IDPs)-->
+
+To get started you'll need to set up an identity provider (if you don't use one already). You will then provide your metadata, either via an XML file or directly using a **Metadata URL** in order to connect to Contrast.
+
+
+---
+
+## EOP
+
+For Enterprise-On-Premises customers, Single Sign-On is configured at the System level by the SuperAdmin.
+
+### Notes About Switching to SSO
+
+If a System Admin decides to set up SSO, it is important to note the following:
+
+Contrast does not support user provisioning, therefore all users must have an existing account in TeamServer before switching the authentication method. Additionally, if users are identified with a user ID rather than an email address, those accounts won’t automatically transfer over to the SSO configuration and will need to be recreated.
```

Comment:
> therefore all users must have an existing account in TeamServer before switching the authentication method. 

Maybe something more like "an existing account or one created in TeamServer before..." Accounts can be created with SSO auth mode.